src/Buerokratt.Common/Models/ModelTypes.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Buerokratt.Common.Models
+{
+    public static class ModelTypes
+    {
+        /// <summary>
+        /// Content type for a message specifying that a routing error has occurred.
+        /// </summary>
+        public static readonly string Error = "application/vnd.dmr.error+json;version=1";
+
+        /// <summary>
+        /// Content type for requesting classification.
+        /// </summary>
+        public static readonly string ClassificationRequest = "application/vnd.classifier.classification+json;version=1";
+
+        /// <summary>
+        /// Content type for routing a message via the DMR.
+        /// </summary>
+        public static readonly string MessageRequest = "application/vnd.dmr.messagerequest+json;version=1";
+
+        /// <summary>
+        /// Content type for indicating acknowledgement of receiving a message. This content type is to indicate that the message in the payload is not one that requires any kind of reply.
+        /// This is a type that is used by Mock Bots only. 
```

Comment:
Is this comment necessary? It may only be used by mock bots now but in the future it cold be used by other participants

Reply:
This is a good question. With real bots, they will always send a "MessageRequest" model type when they're responding to a message. There is an argument to remove this model type from here entirely and just keep it in the MockBot codebase but I was worried that it might confuse things.

Reply:
The bit I was referring to was just the second line "This is a type that is used by Mock Bots only.". I think model is fine, but the comment that seems to suggest it can only be used by mock bot seems unnecessary

Reply:
Maybe this model type is best housed within the mockbot itself if this is a mockbot only thing.  Only it will be sending and receiving these models.  Maybe it's not common enough to go in this library.

Reply:
Sounds good. I've removed it and this will feature in a PR in the MockBot repo.   Thanks both 🙏